vpd-tool/include/tool_types.hpp

```diff
@@ -40,5 +40,13 @@ using DbusVariantType = std::variant<
     std::vector<std::tuple<sdbusplus::message::object_path, std::string,
                            std::string, std::string>>
  >;
+
+using Path = std::string;
```

Comment:
I guess this was discussed that we will not have types defined for simple types. Only complex types should be defined for understanding?